modules/console/pages/ui/add-license.adoc

```diff
@@ -0,0 +1,58 @@
+= Manage Enterprise Edition Licenses through Redpanda Console
+:description: Learn how to manage Enterprise Edition licenses in Redpanda Console.
+
+You add, update and check your xref:get-started:licensing/overview.adoc#console[Enterprise Edition license] for both Redpanda and Redpanda Console directly through the Redpanda Console UI.
+
+== Prerequisites
+
+- You must have an Enterprise Edition license. If you don't have a license yet, https://www.redpanda.com/contact[request one^].
++
+If Redpanda Console has enterprise features enabled and it cannot find a valid license either locally or in the connected Redpanda cluster, it shuts down.
+- Redpanda Console must be xref:console:config/connect-to-redpanda.adoc[connected to a Redpanda cluster].
+- Redpanda Console must be xref:console:config/connect-to-redpanda.adoc#admin[configured to connect to the Redpanda Admin API].
+
+TIP: You can also xref:console:config/enterprise-license.adoc[configure Redpanda Console to load the license key from its local configuration].
+
+== Check the license status in Redpanda Console
+
+You can check the expiration date of a license on the Cluster Overview page in Redpanda Console, under the Licensing section.
+
+If the license is due to expire within 30 days, a warning banner is displayed on all pages of Redpanda Console.
+
+See also: xref:get-started:licensing/monitor-license-status.adoc[].
+
+== Upload a new license
+
+When a new license is uploaded through Redpanda Console, it is replicated across the cluster and stored persistently in Redpanda's internal metadata, ensuring it is retained across restarts.
+
+include::console:partial$licensing/kubernetes-note.adoc[]
+
+To upload a new license directly through the Console UI:
+
+. Open the Upload License page, using one of the following methods:
++
+- Cluster Overview page:
++
+Navigate to the Cluster Overview page in Redpanda Console.
+Under the Licensing section, click on the *Upload new license* link to upload a new license key.
+
+- Expiration warning banner:
++
+If the existing license expires soon, you can click the *Upload license* button in the expiration warning banner.
+
+. Upload your license.
++
+image::console:license.png[]
```

Comment:
I found this image a bit confusing. What am I meant to do to upload the license?

Reply:
clarified the instruction.